dashboards/Hub-ECS.json

```diff
@@ -2170,7 +2170,7 @@
       ],
       "targets": [
         {
-          "expr": "node_memory_MemFree / 1024 / 1024 / 1024",
+          "expr": "(node_memory_MemFree + node_memory_Buffers + node_memory_Cached) / 1024 / 1024 / 1024",
```

Comment:
How about node_memory_MemAvailable?

Reply:
For example, `node_memory_MemAvailable / 1024 / 1024 / 1024`

Reply:
TIL. +1 to this suggestion, see also [the man page for free](http://manpages.ubuntu.com/manpages/xenial/en/man1/free.1.html) for more information, in particular:

             Estimation of how much memory is available for starting new  applications,  without
              swapping.  Unlike  the  data provided by the cache or free fields, this field takes
              into account page cache and also that not all  reclaimable  memory  slabs  will  be
              reclaimed  due  to  items being in use (MemAvailable in /proc/meminfo, available on
              kernels 3.14, emulated on kernels 2.6.27+, otherwise the same as free)

Reply:
🉑